.azure/azure-pipelines.ci.yml

```diff
@@ -673,10 +673,10 @@ stages:
     parameters:
       image: windows-2022
       platform: windows
-      tls: openssl
+      tls: schannel
```

Comment:
Schannel EC builds are not built, so more needs to be fixed.

Reply:
I could have sworn I built both... :|